fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2549,7 +2582,7 @@ private void createLakeTablets(LakeTable table, long partitionId, MaterializedIn
         }
 
         PartitionInfo partitionInfo = table.getPartitionInfo();
-        StorageInfo partitionStorageInfo = partitionInfo.getStorageInfo(partitionId);
```

Comment:
Each partition has his own StorageInfo that only include cache info.
Here we should use partition cache info.